package.json

```diff
@@ -16,11 +16,11 @@
   },
   "dependencies": {
     "ast-types": "^0.8.12",
+    "babel": "^5.8.23",
```

Comment:
Actually even better you can depend on `babylon` which is just the parser in babel.